venues/OpenReview.net/Support/supportProcess.js

```diff
@@ -30,7 +30,7 @@ function(){
             values: note.content['Contact Emails'].concat(['OpenReview.net/Support'])
         }
       },
-      signatures: ['OpenReview.net/Support']
```

Comment:
Why are we removing the Support group and using the super user? the process function doesn't work with this group?